### DISCUSSION TITLE
import: can't create a rectangular screenshot

### DISCUSSION THREAD

**Discussion by tomasmueller** [ImageMagick/ImageMagick#3136] [Help]
when i want to create a screenshot offen i can't create a rectangular border area. the command is simple:

`import output.png`

i have attached a picture to make sure what i mean. as you can see the borders are "something" else.

![Auswahl_001](https://user-images.githubusercontent.com/69788141/104999996-0f806400-5a2e-11eb-8422-7e1242195fe7.png)

**Comment by urban-warrior** [comment]
This issue was fixed in modern releases of ImageMagick.  Which version of ImageMagick are you using?  Try the latest release, 6.9.11-58 or 7.0.10-58 and likely you will get expected results.

**Comment by tomasmueller** [comment]
i have version 6.9.10 installed. this is the last version from the ubuntu repository. is there a ppa for version 6.9.11 or 7?

**Comment by urban-warrior** [comment]
We don't track Ubuntu releases.  You'll need to check with their team.  Alternatively, you can install ImageMagick from an RPM or source distribution.  See the website for instructions.
